### ISSUE TITLE
Too many data objects created

### ISSUE THREAD

**Issue by nilei** [bpmn-io/bpmn-js#486]
When I add a new dataObject reference to the diagram, it always creates a new dataObject. As there is no option to configure the references I would recommend the following:

When adding the first reference, create a new dataObject.
When adding a second reference, ask for reference to existing dataObject or create new dataObject.

**Comment by nikku** [comment]
I'd put another option on the table: What about not creating any DataObject at all and allowing the user to change that via the properties panel ([bpmn-js-properties-panel](https://github.com/bpmn-io/bpmn-js-properties-panel))?

**Comment by nilei** [comment]
Does is make sense to add a reference item without a dataObject?

**Comment by nikku** [comment]
Yes, absolutely. A data object behind a visible data object reference is optional.

**Comment by nilei** [comment]
Hi @nikku, sounds okay to me. I assume you say that the reference is the visual representation of a "data object" and would only need to be linked to a dataObject if serialization is required. That's fair enough I think.

:+1: 
Cheers, Nils
